detox/local-cli/detox-test.js

```diff
@@ -140,15 +140,28 @@ function runJest() {
     recordLogs: program.recordLogs,
     takeScreenshots: program.takeScreenshots,
     recordVideos: program.recordVideos,
-  });
+  };
 
-  console.log(command);
+  console.log(printEnvironmentVariables(detoxEnvironmentVariables) + command);
   cp.execSync(command, {
     stdio: 'inherit',
-    env,
+    env: {
+      ...process.env,
+      ...detoxEnvironmentVariables,
+    },
   });
 }
 
+function printEnvironmentVariables(envObject) {
+  return Object.entries(envObject).reduce((cli, [key, value]) => {
+    if (value == null) {
```

Comment:
preferably use _.isEmpty. Empty strings are not valid.

Reply:
I cannot, because `_.isEmpty(0)` will return true as well.
I'll simply expand this check.